blockchain/tx_pool_test.go

```diff
@@ -1941,45 +1963,45 @@ func TestDynamicFeeTransactionVeryHighValues(t *testing.T) {
 	}
 }
 
-// TODO : Need to KIP-71 hardfork
-//func TestDynamicFeeTransactionHasNotSameGasPrice(t *testing.T) {
-//	t.Parallel()
-//
-//	pool, key := setupTxPoolWithConfig(eip1559Config)
-//	defer pool.Stop()
-//
-//	// Ensure gasFeeCap is greater than or equal to gasTipCap.
-//	tx := dynamicFeeTx(0, 100, big.NewInt(1), big.NewInt(2), key)
-//	if err := pool.AddRemote(tx); err != ErrTipAboveFeeCap {
-//		t.Error("expected", ErrTipAboveFeeCap, "got", err)
-//	}
-//
-//	// The GasTipCap is equal to gasPrice that config at TxPool.
-//	tx2 := dynamicFeeTx(0, 100, big.NewInt(2), big.NewInt(2), key)
-//	if err := pool.AddRemote(tx2); err != ErrInvalidGasTipCap {
-//		t.Error("expected", ErrInvalidGasTipCap, "got", err)
-//	}
-//}
-
-// TODO : Need to KIP-71 hardfork
-//func TestDynamicFeeTransactionAccepted(t *testing.T) {
-//	t.Parallel()
-//
-//	pool, key := setupTxPoolWithConfig(eip1559Config)
-//	defer pool.Stop()
-//
-//	testAddBalance(pool, crypto.PubkeyToAddress(key.PublicKey), big.NewInt(1000000))
-//
-//	tx := dynamicFeeTx(0, 21000, big.NewInt(1), big.NewInt(1), key)
-//	if err := pool.AddRemote(tx); err != nil {
-//		t.Error("error", "got", err)
-//	}
-//
-//	tx2 := dynamicFeeTx(1, 21000, big.NewInt(1), big.NewInt(1), key)
-//	if err := pool.AddRemote(tx2); err != nil {
-//		t.Error("error", "got", err)
-//	}
-//}
+// func TestDynamicFeeTransactionHasNotSameGasPrice(t *testing.T) {
```

Comment:
Why don't we test those?

Reply:
Actually I just wanted to test these in a function. But It needs to be forked in test function and I don't want to test like that. I would rather check internally doing hardfork well, but can't. So I added `TestDynamicFeeTransactionAcceptedKip71` test case and enabled above function you suggested in this review.
Thanks!